server/src/main/java/de/uftos/services/RoomService.java

```diff
@@ -40,20 +40,16 @@ public RoomService(RoomRepository repository, ServerRepository serverRepository)
   /**
    * Gets a page of entries of the lesson table.
    *
-   * @param pageable     contains the parameters for the page.
-   * @param name         the name filter.
-   * @param buildingName the building name filter.
-   * @param capacity     the capacity filter.
-   * @param tags         the tags filter.
+   * @param pageable contains the parameters for the page.
+   * @param search   the search filter.
+   * @param capacity the capacity filter.
+   * @param tags     the tags filter.
    * @return the page of entries fitting the parameters.
    */
-  public Page<Room> get(Pageable pageable, Optional<String> name,
-                        Optional<String> buildingName, Optional<Integer> capacity,
-                        Optional<String[]> tags) {
-    //currently no capacity filter
+  public Page<Room> get(Pageable pageable, Optional<String> search, Optional<Integer> capacity,
```

Comment:
Wouldn't it be better to remove the capacity parameter if it isn't used?
Why keep it, if you replace all the others?

Reply:
I don't think it makes sense to full text search for a number (open for other opinions though), however there may still be a scenario where you want to filter by capacity as well. In theory this could also become an additional field in the GUI, just having it available through the API certainly makes sense anyways though IMO.